assets/js/app.js

```diff
@@ -53,6 +53,28 @@ let liveSocket = new LiveSocket('/live', Socket, {
   },
 });
 
+window.addEventListener('phx:historical-events', (e) => {
+  const tracesListDiv = document.querySelector(
+    `#${e.detail.trace_list_dom_id}`
+  );
+  let separator = document.querySelector('#separator');
+
+  if (separator) {
+    tracesListDiv.removeChild(separator);
+  } else {
+    separator = document.createElement('div');
+    separator.id = 'separator';
+    separator.innerHTML = `
+      Historical events
+      <div class="border-b h-0 border-primary-100"></div>
```

Comment:
This needs some UI polishing - I was thinking about sth like `unread messages` separator in telegram, but you can try asking Maciej the designer if he could come up with something 
<img width="452" alt="Screenshot 2025-02-24 at 14 00 52" src="https://github.com/user-attachments/assets/294f560e-e8c9-425d-9690-d0e143f7cd54" />